katello_certs_tools/sslToolConfig.py

```diff
@@ -740,23 +728,22 @@ def save(self, d, caYN=0, verbosity=0):
               % (gen_req_distinguished_name(rdn), d['--purpose'],  gen_req_alt_names(d, rdn['CN']))
 
         try:
-            rotated = rotateFile(filepath=self.filename,verbosity=verbosity)
-            if verbosity>=0 and rotated:
-                print "Rotated: %s --> %s" % (os.path.basename(self.filename),
-                                              os.path.basename(rotated))
+            rotated = rotateFile(filepath=self.filename, verbosity=verbosity)
+            if verbosity >= 0 and rotated:
+                print("Rotated: %s --> %s" % (os.path.basename(self.filename),
+                                              os.path.basename(rotated)))
         except ValueError:
             pass
         fo = open(self.filename, 'w')
         fo.write(openssl_cnf)
         fo.close()
-        os.chmod(self.filename, 0600)
+        os.chmod(self.filename, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)